extension/persistence/eclipselink/src/main/resources/META-INF/persistence.xml

```diff
@@ -23,7 +23,7 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd">
 
-  <persistence-unit name="polaris" transaction-type="RESOURCE_LOCAL">
+  <persistence-unit name="polaris-staging" transaction-type="RESOURCE_LOCAL">
```

Comment:
At this point, I don't think it's a good idea to change this without a legitimate release